### ISSUE TITLE
Remove icons in main window

### ISSUE THREAD

**Issue by SegiH** [asbru-cm/asbru-cm#659]
**Is your feature request related to a problem? Please describe.**
I love Asbru-CM but I feel that the UI is cluttered with way too many icons in the main window, some of which are redundant and others are probably not used much and all of these icons take up too much real estate in the main window.

**Describe the solution you'd like**
Add an application level menu with File, Edit, Tools, Help like a standard desktop application and remove the icons on the main UI. Trigger the event handler that was triggered by clicking on these icons from the corresponding menu item instead

**Describe alternatives you've considered**
I have attached a screenshot based on Asbru-CM 6.2.2 with each icon labeled with a number for reference. My suggestion is to remove all of the icons and move them to the application menu like this:

1. New Group - File->New->New Group
2. New Connection - File->New->New Connection
3. Edit this Connection - Delete this icon because it is entirely redundant. You can edit a connection by right clicking on the connection or pressing Ctrl+E.
4. Rename this Node - Delete this icon because it is entirely redundant. You can rename a node by right clicking on the connection or pressing F2.
5. Delete this Node(s) - Delete this icon because it is entirely redundant. You can get delete a node by right clicking on the connection. The wording of the icon tooltip text suggests that you can delete multiple nodes but I doubt that most people have a real world case for mass deleting nodes that will have an impact on them (i.e deleting hundreds of nodes at once seems unlikely).
6. Search - Edit ->Search which shows the search panel when selected.
7.  Connect - Remove this icon. You can initiate a connection by double clicking on the connection name. This is entirely useless.
8. Start a new connection without saving it -File->New->New connection without saving it.
9.  Add to/remove from favorites - Add "Add to/remove from favorites" to context menu when right clicking on a connection because this is a connection specific option.
10.  Clusters - Tools->Clusters
11. Scripts -  Tools-> Scripts
12. PCC -  Tools-> PCC
13. Show Hide connections Panel - View->Show connections panel as a checkable menu item
14. Wake On LAN -> Tools->Wake On LAN
15. Launch Local Shell - Tools->Launch Local Shell
16. Preferences - Tools->Preferences
17. Auto Saving Active - File->Save
18. Protect With Password - Tools->Protect with Password
19. About Dialog - Help->About 
20. Quit - File->Exit

**Additional context**
I am a developer and would be glad to help implement this. I am not as experienced in Perl as I am with other languages but I am sure that I could help create a prototype of this request if someone can help me by pointing me to the right resource to edit to modify the main UI window.

**Comment by SegiH** [comment]
![AsbruCM](https://user-images.githubusercontent.com/3309961/86270698-7d436c00-bb80-11ea-94fa-8b7883521b55.PNG)

**Comment by gfrenoy** [comment]
Thanks a lot for the suggestion and your offer for help, Sergi !  Glad to read you like Ásbrú and wanted to actively contribute, this is really the purpose of this project :)

I also believe the UI needs a redesign and I will certainly be happy to let you know what you need to know to move forward with this activity !

Now, it's certainly not an easy one and we need to take the necessary step backward to carefully think about what we want to do.  This will also have a big impact on the code so we might need to dedicate one milestone / branch for that change.

My personal feeling would be that we should not remove *everything* to menus.  Hence we should keep some icons/buttons. 
 Maybe we should keep one action bar somewhere (don't know where...) ; have the most useful actions there and maybe leave the user customize it ?  And all option options would be in a "burger menu".  Would the default GNOME [files browser](https://wiki.gnome.org/action/show/Apps/Files) be an inspiration ?

If I look at my own behavior, I do often use some of the buttons and I would not like to make one or more additional clicks to access those features:

- add a new connection / folder
- edit the selection connection
- preferences
- local shell
- quit

For all the other features, I'm using the shortcut keys or a right-click to access the context menu.

I also wanted to mention that - thanks to Hans ;p -  there is a "Compact" mode:
![image](https://user-images.githubusercontent.com/981546/86510081-0f4fac80-bded-11ea-9d4e-e37e25708582.png)

This mode makes the Ásbrú user experience quite different ; especially for small screens.

All in all, I'm wondering if we should not start by organizing a discussion to decide about what we want to do exactly and let you know the basics ; let me send you an email to arrange this.

Thanks again !

**Comment by SegiH** [comment]
> Thanks a lot for the suggestion and your offer for help, Sergi ! Glad to read you like Ásbrú and wanted to actively contribute, this is really the purpose of this project :)
> 
> I also believe the UI needs a redesign and I will certainly be happy to let you know what you need to know to move forward with this activity !
> 
> Now, it's certainly not an easy one and we need to take the necessary step backward to carefully think about what we want to do. This will also have a big impact on the code so we might need to dedicate one milestone / branch for that change.
> 
> My personal feeling would be that we should not remove _everything_ to menus. Hence we should keep some icons/buttons.
> Maybe we should keep one action bar somewhere (don't know where...) ; have the most useful actions there and maybe leave the user customize it ? And all option options would be in a "burger menu". Would the default GNOME [files browser](https://wiki.gnome.org/action/show/Apps/Files) be an inspiration ?
> 
> If I look at my own behavior, I do often use some of the buttons and I would not like to make one or more additional clicks to access those features:
> 
>     * add a new connection / folder
> 
>     * edit the selection connection
> 
>     * preferences
> 
>     * local shell
> 
>     * quit
> 
> 
> For all the other features, I'm using the shortcut keys or a right-click to access the context menu.
> 
> I also wanted to mention that - thanks to Hans ;p - there is a "Compact" mode:
> ![image](https://user-images.githubusercontent.com/981546/86510081-0f4fac80-bded-11ea-9d4e-e37e25708582.png)
> 
> This mode makes the Ásbrú user experience quite different ; especially for small screens.
> 
> All in all, I'm wondering if we should not start by organizing a discussion to decide about what we want to do exactly and let you know the basics ; let me send you an email to arrange this.
> 
> Thanks again !

GFrenoy. I have an unrelated question that I wanted to ask you about Asbru but don't have your email address any more. I wanted to ask you if you would care or be interested in an simple app that I wrote for Windows which lets you run Asbru-CM on Windows 10 without any background console windows open.It does rely on you to have already installed WSL/WSL2 + Ubuntu image from MS Store + XMing. 

If you're interested, let me know and I will put it up on Github

**Comment by gfrenoy** [comment]
>  I have an unrelated question

Yes, please create a new issue :)  Please follow up in #695, thanks :)
